modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,278 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+image::cloudhub-dlb-mapping-rules-reorder.png[Handle to use to reorder a mapping rule]
+. Grab the handle to move the rule up or down in the list.
+. Click *Done Editing*.
 
-You can define both input and output URLs using patterns or literal strings.
+=== Change the Priority of Mapping Rules Using the CLI 
 
-Mapping rules are attributes of the load balancer's SSL endpoint, which is identified by the certificate name.
-When you create a mapping rule, you must specify a certificate CN. Omitting the `[certificateName]` parameter adds the mappings to the default endpoint.
+To specify the mapping rule priority when creating a mapping rule with the CLI, use the `index` value.
+The rule at index `0` has the highest priority. The higher the index assigned, the lower the priority.
 
-If your SSL endpoint sets a https://en.wikipedia.org/wiki/Wildcard_certificate[wildcard certificate], and you want to use the subdomain portion in a mapping rule, you can use the pre-defined `{subdomain}` variable.
+See xref:anypoint-platform-cli-commands.adoc#cloudhub-load-balancer-mappings-add[cloudhub load-balancer mappings add].
 
-The rule which is defined first has high priority against other ones defined after it. This means that the first matched rule will be applied.
-You can create, view and delete existing rules using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-add[`mappings add`], xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-describe[`mappings describe`] and xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-remove[`mappings remove`] commands respectively.
+=== Change the Priority of Mapping Rules Using the CloudHub API 
 
+When using the API to create a mapping rule, you cannot specify a priority order. 
+However, you can update the order of the rules later by sending a `PATCH` request to the load balancer endpoint:
 
-== Mapping Rule Examples
+[source,text,linenums]
+----
+anypoint.mulesoft.com/cloudhub/api/organizations/{orgid}/loadbalancers/{loadbalancerId}
+----
 
-The external load balancer domain name depends on the unique name you assign to it. In the following mapping rule examples, the load balancer is `lb-demo`.
+To determine the load balancer ID, perform a `GET` request to your endpoint:
 
-By default, your load balancer listens to external requests on HTTPS and communicates internally with your worker over HTTP. If you configured your Mule application within the VPC to listen on HTTPS, make sure you set `upstreamProtocol` to `HTTPS` when creating the mapping list using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-add[`load-balancer mappings add`] command.
+[source,text,linenums]
+----
+/organizations/{orgid}/loadbalancers
+----
 
-=== URL Mapping
 
-You can pass the app name as an input URI and map it directly to the app name in CloudHub:
+== Using a Literal 1:1 Mapping
+
+If you have only one application, you can map the literal app name.
```

Comment:
Remove or put it as an example. It currently read as it would be possible to do ONLY if you have 1 input URI mapped to an application. 

You can also do literal 1:1 mappings for multiple applications by changing the InputURI to be static instead of a defined pattern:
0 /v1/applicationX    application-x http
1 /v1/applicationY    application-y http
2 /v2/applicationx    application-x-v2

Reply:
OK, take a look now and let me know if I messed up :)

Reply:
I don't see a difference
`In this case, the rule must match the application exactly. 

This rule maps the default load balancer `lb-demo.lb.anypointdns.net` directly to the `myApp.cloudhub.io` app in CloudHub:

[%header,cols="10a,20a,20a,20a,10a"]
|===
| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
| 1 | / | myApp | / | http 
|===

This rule matches the `dev-misc-mythellotest-v1-myapp` app:`

Reply:
It used to say:
>If you have only one application, you can map the literal app name.
It now says:
> Instead of using a pattern, you can map the literal app name.
> In this case, the rule must match the application exactly.

If this is incorrect, please let me know how can I correct it.

Reply:
I'd do 

> Instead of using a pattern, you can map literal app names.
In this case, the rules must match the applications exactly.

Again, not a single app for the static mapping (1:1). You can do a single app by mapping / but that's not the same as saying you can do 1:1 mappings.

Reply:
OK, replaced.